tox.ini

```diff
@@ -33,6 +33,13 @@ deps = cmd_examples: {[_cmd_examples]deps}
        build_examples: {[_build_examples]deps}
        flakes: {[_flakes]deps}
        all: {[_all]deps}
+       nose
+
+commands = nosetests -vv --nologcapture --with-doctest --with-coverage --cover-package=pyct
```

Comment:
pyct previously had no unit tests. You have added unit tests: great :) The tox.ini format is no doubt tricky: what you need to do is add a "unit tests" section with this command.

See e.g. https://github.com/holoviz/panel/blob/master/tox.ini

![Screenshot from 2020-05-07 07-37-31](https://user-images.githubusercontent.com/1929/81262476-d6d26300-9035-11ea-9cdb-2b16a040e1e2.png)

![Screenshot from 2020-05-07 07-37-49](https://user-images.githubusercontent.com/1929/81262478-d76af980-9035-11ea-8af2-7436014897e6.png)

![Screenshot from 2020-05-07 07-38-01](https://user-images.githubusercontent.com/1929/81262480-d8039000-9035-11ea-859f-fb0f0573334d.png)

![Screenshot from 2020-05-07 07-38-17](https://user-images.githubusercontent.com/1929/81262482-d8039000-9035-11ea-9e6d-34c93f92af80.png)

![Screenshot from 2020-05-07 07-38-31](https://user-images.githubusercontent.com/1929/81262488-d89c2680-9035-11ea-9224-152accc13ddc.png)


Yes it's ridiculous.